js/src/ui/adminBar.js

```diff
@@ -1,14 +1,29 @@
 /**
- * Updates the traffic light present on the page
+ * Updates the admin bar for the page.
  *
  * @param {Object} indicator The indicator for the keyword score.
+ * @param {string} keyword The focus keyphrase for the post.
  *
  * @returns {void}
  */
-function updateAdminBar( indicator ) {
+function updateAdminBar( indicator, keyword ) {
+	// Updates the traffic light
 	jQuery( ".adminbar-seo-score" )
 		.attr( "class", "wpseo-score-icon adminbar-seo-score " + indicator.className )
 		.find( ".adminbar-seo-score-text" ).text( indicator.screenReaderText );
+
+	// Updates the focus keyhprase in the menu
+	const keywordMenuElement = jQuery( "#wp-admin-bar-wpseo-focus-keyphrase" );
+
+	if ( keyword === "" ) {
+		keywordMenuElement.hide();
+	}
```

Comment:
Minor: this will hide the keyphrase element, while on the front end users will see "not set". Maybe worth considering a consistent UI:

<img width="450" alt="Screenshot 2019-10-09 at 11 08 59" src="https://user-images.githubusercontent.com/1682452/66467880-7a9a6700-ea85-11e9-9f5a-2e2b03c309c7.png">